scoring.js

```diff
@@ -31,14 +30,13 @@ calculateScore = (player) => {
             (puntReturn.yards / 15) +
             (puntReturn.touchdowns * 6) +
             (puntReturn.fumbles * -3)
+  } else if (player.position === 'WR') {
+    let rushing = player.stats.rushing
+    let receiving = player.stats.receiving
```

Comment:
Switch statement good last case should be your default and it is standard to name it that.